client/src/app/behaviors/update-times-interval.coffee

```diff
@@ -0,0 +1,14 @@
+$ = require('jquery')
+Marionette = require('backbone.marionette')
+
+module.exports = Marionette.Behavior.extend(
+
+  onRender: ->
+    @view.statusInterval = setInterval ( => @updateCollection() ), 1000 * 60 * 1
```

Comment:
That was more for the sake of clarity, in terms of how many minutes the interval is set for.  It can be taken out though.